lib/constants/index.ts

```diff
@@ -1,2 +1,3 @@
 export * from './platforms';
 export * from './programming-language';
+export * from './category';
```

Comment:
```suggestion
export type { Category } from './category';
```
otherwise we have a runtime import.
maybe we should move this to lib/types instead.

Reply:
I think we should keep it in constants as we use it more like an enum instead of a type.